pkg/store/proxy.go

```diff
@@ -294,7 +294,7 @@ func (s *ProxyStore) Series(r *storepb.SeriesRequest, srv storepb.Store_SeriesSe
 		level.Debug(s.logger).Log("msg", strings.Join(storeDebugMsgs, ";"))
 		if len(seriesSet) == 0 {
 			// This is indicates that configured StoreAPIs are not the ones end user expects.
-			err := errors.New("No store matched for this query")
+			err := errors.New("No available StoreAPIs matched for this query")
```

Comment:
Thanks for that! But I disagree this change. It's still confused when the stores are all available(IMO that means all of stores are healthy in the Stores page).

Reply:
It includes all case in my comment in the commit i think.

Reply:
Yes, but I agree with @jojohappy 

It feels like available can be confusing. What about: 
```suggestion
			err := errors.New("No StoreAPI matched for this query")
```